intermediate-python-debugging/exercise-using-debugger.md

```diff
@@ -0,0 +1,76 @@
+# Problem Set: Using the Debugger
+
+In this exercise we will use the debugger to identify two errors.  To run the activity clone the code at [AdaGold/python-debugging](https://github.com/AdaGold/python-debugging).  Then set up the project by creating a virtual environment and installing pytest.
+
+The code contains two bugs, one runtime error crashes the program on execution and the other is a logic error which calculates the incorrect sum for an Order instance.  
```

Comment:
This comes across as a little unclear to me.

I think if you convert this to an [Activity or Problem Set template](https://github.com/Ada-Developers-Academy/core/wiki/Templates-and-Stubs), this content would split underneath the different headers in the template easily.

I think you can also cut the part about setting up the project with the venv and installing pytest, and instead in the instructions point them to the repo README and say "Setup the activity by visiting here and following the instructions on the README."

I'd also love an explicit step that says something like "After following the setup instructions on the python-debugging repo, follow the instructions below."